versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/AbstractTracing.java

```diff
@@ -141,7 +141,7 @@ public boolean equals(Object o) {
       if (this == o) {
         return true;
       }
-      if (o == null || getClass() != o.getClass()) {
```

Comment:
kind of annoying that errorprone complains about generate equals() methods

Reply:
yea :(